app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -116,23 +119,33 @@ import javax.inject.Singleton
   qualifiers = "port-xxhdpi"
 )
 class ExplorationActivityTest {
-  private lateinit var networkConnectionUtil: NetworkConnectionUtil
-  private lateinit var explorationDataController: ExplorationDataController
+
+  @Inject
+  lateinit var explorationDataController: ExplorationDataController
+
+  @Inject
+  lateinit var networkConnectionUtil: NetworkConnectionUtil
 
   @Inject
   lateinit var context: Context
 
+  @Inject
+  lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
+
   private val internalProfileId: Int = 0
 
   @Before
   fun setUp() {
     Intents.init()
     setUpTestApplicationComponent()
+    context = ApplicationProvider.getApplicationContext()
```

Comment:
As we are injecting the context above we don't need to initialize it. Whenever we `@Inject` something it seems that will be provided by dagger and we don't need to initialize it.